jina/types/sets/querylang.py

```diff
@@ -85,4 +85,5 @@ def build(self):
         """Build a doc_id to doc mapping so one can later index a Document using
         doc_id as string key
         """
-        self._docs_map = {d.id: d for d in self._querylangs_proto}
+        # TODO This is a temp fix, QueryLangProto do not have an id field.
+        self._querylangs_map = {d.name: d for d in self._querylangs_proto}
```

Comment:
Please fix the docstrings as well
```suggestion
        # TODO This is a temp fix, QueryLangProto do not have an id field.
        self._querylangs_map = {q.name: q for q in self._querylangs_proto}
```